ipamd/datastore/data_store.go

```diff
@@ -372,17 +372,17 @@ func (ds *DataStore) FreeENI() string {
 		return ""
 	}
 
-	ds.total -= len(ds.eniIPPools[deletableENI.ID].IPv4Addresses)
-	ds.assigned -= deletableENI.AssignedIPv4Addresses
-	log.Infof("FreeENI %s: IP address pool stats: free %d addresses, total: %d, assigned: %d",
-		deletableENI.ID, len(ds.eniIPPools[deletableENI.ID].IPv4Addresses), ds.total, ds.assigned)
-	deletedENI := deletableENI.ID
-	delete(ds.eniIPPools, deletableENI.ID)
+	removedENI := deletableENI.ID
+	perENIIPAddressCount := len(ds.eniIPPools[removedENI].IPv4Addresses)
```

Comment:
Why per? It would be more like removed if you go that way or just deletableENIPAddressCount.

Reply:
How about just `eniIPCount`?